cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go

```diff
@@ -104,24 +104,31 @@ func TestValidateTokenGroups(t *testing.T) {
 	}
 }
 
-func TestValidateNodeName(t *testing.T) {
+func TestValidateNodeRegistrationOptions(t *testing.T) {
 	var tests = []struct {
-		s        string
-		f        *field.Path
-		expected bool
+		nodeName       string
+		criSocket      string
+		expectedErrors bool
 	}{
-		{"", nil, false},                 // ok if not provided
-		{"1234", nil, true},              // supported
-		{"valid-nodename", nil, true},    // supported
-		{"INVALID-NODENAME", nil, false}, // Upper cases is invalid
+		{"", "/some/path", true},                                                      // node name can't be empty
```

Comment:
These should really have test names and be run as sub tests (`t.Run(...`)

Reply:
If I have time I'll do that, otherwise just leave up to a more general refactor of our tests